src/matter/cluster/server/LevelControlServer.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { LevelControlCluster } from "../LevelControlCluster";
+import { ClusterServerHandlers } from "./ClusterServer";
+
+/*
+TODO:  not clear about how parameters get passed and returned. And there is logic
+that needs to be added for changing other clusters from the WithOnOff commands and to
+set attributes other than currentLevel
+
+The moveToLevel commands have been tested in Google Home and in Alexa
+*/
+
+export const LevelControlClusterHandler: () => ClusterServerHandlers<typeof LevelControlCluster> = () => ({
```

Comment:
Honestly I would implement all functions the same for now ... all with a "set" of the level without more login and add todos to add that later, also add later the onoff but for now set level

Reply:
Have all the functions in, but only the "Move to Level" commands include a target level. The others are along the line of "move current level" with a time limit. Should we throw an "Unimplemented" error on any command that is just stubbed out?

Also have thoughts in the "TODO" comments regarding how much logic should be in the library/command vs. in the caller of the library since the library has no knowledge of the underlying hardware capabilities. Do we need some sort of callback from the library?